composer_migration/src/composer_migration/__main__.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import logging
+import subprocess
+import re
+import tempfile
+import os
+import sys
+
+from composer_migration.lib.strategies.migrate_kubernetes_pod_operator_ast import (
+    CheckKubernetesPodOperator,
+)
+from composer_migration.lib.comparator import DAGComparator
+
+
+def run_checks(gcp_project: str, composer_environment: str, location: str) -> None:
+    # get bucket where dags are stored
+    try:
+        composer_env_info = subprocess.run(
```

Comment:
I had lots of problems with this approach. I ended up having a much easier time with the python client library for composer. 

This is how I imported it: https://github.com/GoogleCloudPlatform/composer-utilities/blob/main/cloudcomposerdiff/src/cloudcomposerdiff/__main__.py#L18

This is how I called the API:
https://github.com/GoogleCloudPlatform/composer-utilities/blob/main/cloudcomposerdiff/src/cloudcomposerdiff/__main__.py#L81-L83

As per the docs you should be able to find dags_gcs_prefix in the response: https://cloud.google.com/python/docs/reference/composer/latest/google.cloud.orchestration.airflow.service_v1.types.EnvironmentConfig

Reply:
great call - I wrote this part of it long before I read your util. I will incorporate that change!